client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationCounty.tsx

```diff
@@ -45,6 +45,8 @@ const TransferInvestigationCounty = (props: Props) => {
             }).then((result) => {
                 if (result.value) {
                     onConfirm(data[TransferInvestigationInputsNames.COUNTY], data[TransferInvestigationInputsNames.REASON] || '');
+                    alertSuccess('החקירות הועברו בהצלחה');

```

Comment:
I'd make a function in the footer which is sent here by props because it is similar in both transfer dialogs